charts/csi-baremetal-deployment/templates/csi-baremetal_v1_deployment.yaml

```diff
@@ -47,9 +47,6 @@ spec:
           pullPolicy: {{ .Values.image.pullPolicy }}
           tag: {{ .Values.driver.drivemgr.image.tag | default .Values.image.tag }}
         endpoint: {{ .Values.driver.drivemgr.grpc.server.endpoint }}
-        deployConfig: {{ .Values.driver.drivemgr.deployConfig }}
```

Comment:
Those values need to be used for loopbackmgr ConfigMap

Reply:
I left this field in helm-values and ConfigMap is deployed, when it is set. Does operator also use it and we need it in CR?

Reply:
I was talking about these changes https://github.com/dell/csi-baremetal-operator/pull/13/files#diff-0f1052548ad23899529824f0a163e27c090309357863bf11b874386c2fbb796fL48.
As I saw, you removed it from helm values. But it's default values for ConfigMap, I suppose you shouldn't do it.

Reply:
Got it, fixed